kratos/processes/tetrahedral_mesh_orientation_check.cpp

```diff
@@ -192,7 +192,11 @@ void TetrahedralMeshOrientationCheck::Execute()
 
     //check that all of the conditions belong to at least an element. Throw an error otherwise (this is particularly useful in mpi)
     for (auto& r_cond : mrModelPart.Conditions()) {
-        KRATOS_ERROR_IF(r_cond.IsNot(VISITED)) << "Found a condition without any corresponding element. ID of condition = " << r_cond.Id() << std::endl;
+        const GeometryType& r_geometry = r_cond.GetGeometry();
+        const GeometryData::KratosGeometryType geometry_type = r_geometry.GetGeometryType();
+        if (geometry_type == GeometryData::KratosGeometryType::Kratos_Triangle3D3 || geometry_type == GeometryData::KratosGeometryType::Kratos_Line2D2) {
```

Comment:
Does this process work with quadratic elements or only linear? If so, we should add quadratic geometries here also

Reply:
I agree

Reply:
Instead of geoemtry type there is something more generic that includes all the geometries of the same kind (triangles, quads)

Reply:
the idea of this MR is to to avoid flagging as errors line 3d conditions in 3d tetra meshes. In these cases, the lines will not be part of any "face" (edge) of a 2d tria.

as for making it compatible with quadratic , yes , i think it could be done. (may i do it in a different mr?)

Reply:
My point was that if the process was already working with quadratic geometries, we should consider that in the check. If the process only works with linear geometries now, we can leave it as this for the moment.